hapi-utils/src/test/java/com/hedera/services/sysfiles/serdes/FeesJsonToProtoSerdeTest.java

```diff
@@ -30,16 +30,17 @@
 import java.lang.reflect.Constructor;
 import java.lang.reflect.InvocationTargetException;
 import java.nio.file.Paths;
-import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.*;
```

Comment:
Can we use `org.junit.jupiter.api.Assertions;` instead of `org.junit.jupiter.api.*` . I think `./gradlew spotlessApply` should do that.

Reply:
Sure thing. I wasn't sure if intellij did this or spotless. I'll make sure it isn't a star import